Description: Correct pod NAME issues in LedgerSMB/CancelFurtherProcessing.pm
 Does not have the dash and short description in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472543&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/CancelFurtherProcessing.pm
+++ b/LedgerSMB/CancelFurtherProcessing.pm
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-CancelFurtherProcessing
+LedgersSMB::CancelFurtherProcessing - A simple wrapper around Error.pm for error trapping.
 
 =head1 SYNOPSIS
 
